README.md

```diff
@@ -29,7 +35,7 @@
 
 ## Architecture
 
-`Falco Talon` can receive the `events` from [`Falco`](https://falco.org) or [`Falcosidekick`](https://github.com/falcosecurity/falcosidekick):
+`Falco Talon` can receive the `events` from [`Falco`](https://falco.org) or [`Falcosidekick`](https://github.com/falcosecurity/falco-talon):
```

Comment:
Pointing to Talon here instead of Sidekick

Reply:
Nice catch, will fix it later. ThankS